web-ng/src/app/components/drawing-tools/drawing-tools.component.css

```diff
@@ -27,3 +27,15 @@
   background-color: white;
   transform: translate(-50%, 0);
 }
+
+.add-point-button {
+  background-color: #202225;
```

Comment:
Can these colors be refactored into styles/_variables.scss instead of being declared here?

Reply:
Good idea, extracted as css variables.